notes/src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Note from "./components/note";
```

Comment:
Let's separate imports by their types (external and internal)
```suggestion
import React from "react";

import Note from "./components/note";
```